package.json

```diff
@@ -46,7 +46,7 @@
 	"dependencies": {
 		"@ardenthq/arkvault-crypto": "^0.0.7",
 		"@ardenthq/arkvault-url": "^1.2.0",
-		"@arkecosystem/typescript-client": "^0.0.6",
+		"@arkecosystem/typescript-client": "^0.0.9",
```

Comment:
updates typescript client so the `unconfirmed` methods use the right underlying API to avoid the previously used workaround